admin_view_permission/admin.py

```diff
@@ -100,10 +119,14 @@ def get_fields(self, request, obj=None):
         if ((self.has_view_permission(request, obj) and (
             obj and not self.has_change_permission(request, obj, True))) or (
                 obj is None and not self.has_add_permission(request))):
-            fields = super(AdminViewPermissionBaseModelAdmin, self).get_fields(
-                request, obj)
+            fields = super(
+                AdminViewPermissionBaseModelAdmin,
+                self).get_fields(request, obj)
+            excluded_fields = self.get_excluded_fields()
```

Comment:
In Django 1.11 the function `get_exclude` is introduced. What is your opinion to make a wrapper which executes the `get_exclude` for Django 1.11 and the `get_exclude_fields` for the other versions?